response_operations_ui/__init__.py

```diff
@@ -149,5 +153,27 @@ def user_loader(user_id):
     Session(app)
 
     setup_blueprints(app)
+    setup_oidc(app)
 
     return app
+
+
+class MissingEnvironmentVariable(Exception):
+    pass
+
+
+def setup_oidc(application):
+    def client_ids_exist():
+        if not application.config.get("CIR_OAUTH2_CLIENT_ID"):
+            raise MissingEnvironmentVariable("CIR_OAUTH2_CLIENT_ID Missing")
+
+    if not (oidc_token_backend := application.config.get("OIDC_TOKEN_BACKEND")):
+        raise MissingEnvironmentVariable("OIDC_TOKEN_BACKEND Missing")
+
+    if oidc_token_backend == "gcp":
+        client_ids_exist()
+        application.oidc["oidc_credentials_service"] = OIDCCredentialsServiceGCP()
+    elif oidc_token_backend == "local":
+        application.oidc["oidc_credentials_service"] = OIDCCredentialsServiceLocal()
+    else:
+        raise NotImplementedError("Unknown OIDC_TOKEN_BACKEND")
```

Comment:
Does what it says on the tin. Allows us to run this locally without any GCP dependencies on their OIDC service layer.